### PR TITLE
Mobile protofolio

### PR DESCRIPTION
**Pull request by Abdo9826**
setup and mobile version skeleton with HTML and CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<nav>
+   <div>
+       <p>9:41</p>
+       <div class="ul">
+        <img src="image/CellularConnection.png" alt=""> 
+       <img src="image/Wifi.png" alt=""> 
+        <img src="image/Battery.png" alt="">
+       </div>
+
+</div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly remove this div as it is the header of the phone and you shouldn't design it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+<div>
+<h4>Abdo</h4>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly put this in an `a` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+<P class="img"><img src="image/Union.png" alt=""></P>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Please be noticed that only the hamburger button is the one that should be inside the `nav` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            <section>
+                <div><img src="image/Vector.png" alt=""></div>
+                <div><img src="image/linkedin.png" alt=""></div>
+                <div><img src="image/tw.png" alt=""></div>
+                <div><img src="image/gmail.png" alt=""></div>
+
+            </section>
+
+   
+        </header>
```
Comment (ShahierNashaat) [inline index.html:35]: - [x] Kindly be noticed that only the logo and the hamburger button are the ones that should be inside the `header` tag.

File: a.css [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: absolute;
+  margin-left: 33%;
+}
```
Comment (ShahierNashaat) [inline a.css]: - [ ] Please make your design responsive.
Screenshot from 2022-04-21 21-54-00

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Abdo9826 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Please add the setup steps in your readme file.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — COMMENTED** [review]
Hi @Abdo9826 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the Linter result is not green ✔️ 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html (hunk 4 above)
Comment (Munsa1) [reply index.html:35]: This requested change has not been implemented, Kindly remove lines 38 to 58 from the `header` tag and create a `section` below the header where you will paste that code 
e.g
image

- The header should remain looking something like this
image


[Optional] Note to indent your code properly as this helps find bugs easily and makes your code more readable to other collaborators.

File: a.css [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+  width: 375px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: absolute;
+  margin-left: 933px;
```
Comment (Munsa1) [inline a.css]: Again as requested by the previous reviewer your page is not responsive. Kindly remove that left margin inline `16` as it is pushing your whole page out of view in mobile view.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+<nav>
+   
+       <p>9:41</p>
+       <div class="ul">
+        <img src="image/CellularConnection.png" alt=""> 
+       <img src="image/Wifi.png" alt=""> 
+        <img src="image/Battery.png" alt="">
+       </div>
+       <P class="img"><img src="image/Union.png" alt=""></P>
+
```
Comment (Munsa1) [inline index.html]: As requested by the previous reviewer, these Icons are not part of the Figma template rather they just serve to show that the site is being viewed on a phone. Kindly remove them and leave the only the logo_name and the mobile menu_bar

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Abdo9826, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- linters passing
- Correct gitflow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Making your readme file more descriptive by adding a description of the project

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Abdo9826 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description

## Optional suggestions

- [ ] Kindly make sure your page is responsive, currently, it works but it has a fixed width which is not a good design.
- [ ] Please consider adding more content in your `readme` file, especially in the Getting Started section and acknowledgement section.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me using @danmainah in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**